filecoin-proofs/src/constants.rs

```diff
@@ -156,6 +153,9 @@ pub const MINIMUM_RESERVED_BYTES_FOR_PIECE_IN_FULLY_ALIGNED_SECTOR: u64 =
 /// The minimum size a single piece must have before padding.
 pub const MIN_PIECE_SIZE: UnpaddedBytesAmount = UnpaddedBytesAmount(127);
 
+/// The maximum number of challenges per partition the circuits can work with.
+pub(crate) const MAX_CHALLENGES_PER_PARTITION: u8 = 18;
```

Comment:
To me, the word `MAX` implies that a PoRep circuit can prove less than 18 challenges, whereas a name like `POREP_CHALLENGES_PER_PARTITION` would imply that a PoRep circuit can prove exactly 18 challenges.

Reply:
Yes, that's the reason I used `MAX`, as less challenges also seem to work.